ui/raidboss/data/07-dt/raid/r2s.ts

```diff
@@ -102,10 +104,27 @@ const triggerSet: TriggerSet<Data> = {
       response: Responses.sharedTankBuster(),
     },
     {
-      id: 'R2S Headmarker Cone Tankbuster',
+      id: 'R2S Headmarker Cone Tankbuster Collect',
       type: 'HeadMarker',
       netRegex: { id: headMarkerData.tankLaser, capture: true },
-      response: Responses.tankCleave(),
+      run: (data, matches) => data.tankLaserCollect.push(matches.target),
+    },
+    {
+      id: 'R2S Headmarker Cone Tankbuster',
+      type: 'HeadMarker',
+      netRegex: { id: headMarkerData.tankLaser, capture: false },
+      delaySeconds: 0.1,
```

Comment:
If I'm reading this right, this will also need a `suppressSeconds` property here, since otherwise this would activate twice, once for each head marker.

Reply:
Ah yes with the change to two triggers we need to suppress multiple reports. I think we likely need to find a couple of my other PRs the same way.

Reply:
That or add a check against the array size or something. But I think the suppressSeconds is the best solution

Reply:
LGTM